packages/commonwealth/server/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
```

Comment:
Don't we already have a server tsconfig in the package/commonwealth level?

Reply:
yes, no sure why this dup has been created, its gone now